iml-agent/src/cli.rs

```diff
@@ -607,12 +629,28 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
                     action,
                 };
 
-                if action_cloudsync::process_fids((llapi.mntpt(), task_args, fidlist))
-                    .await
-                    .is_err()
-                {
-                    tracing::error!("Filesync failed");
-                    exit(exitcode::IOERR);
+                let result =
+                    action_cloudsync::process_fids((llapi.mntpt(), task_args, fidlist)).await;
+                match result {
+                    Ok(reslist) => {
+                        if reslist.is_empty() {
+                            println!("Ok!");
+                            exit(0);
+                        }
+
+                        for err in reslist.iter() {
```

Comment:
How large might this list be?

Reply:
As many items as the user typed in (not more)